app/views/simulator/edit.html.erb

```diff
@@ -714,6 +714,15 @@
   <% end %>
 </script>
 
+<!-- report an issue validator -->
+<script>
+  function validator() {
+    if (document.getElementById("issuetext").value.length == 0) {
+      alert("issue cannot be empty")
```

Comment:
Please change the prompt alert message.
`issue cannot be empty` -> `Description cannot be empty! `

Reply:
> Please change the prompt alert message. `issue cannot be empty` -> `Description cannot be empty! `

done!